src/migrations/201602231400_initial.js

```diff
@@ -17,7 +17,7 @@ function createAccountsTable (knex) {
 function createFulfillmentsTable (knex) {
   return knex.schema.createTableIfNotExists('fulfillments', (table) => {
     table.increments()
-    table.uuid('transfer_id').unique().index()
+    table.uuid('transfer_id').unique()
```

Comment:
If I include `.index()` here, oracle complains: `Knex:warning - migrations failed with error: create index "fulfillments_transfer_id_index" on "fulfillments" ("transfer_id") - ORA-01408: such column list already indexed`. This spirals into other errors that break the migrations. My belief is that `table.index(['subscription_id', 'transfer_id'], 'subscription_transfer')` indexes this column, but I could be wrong, and I'd be happy for input.